micrometer-spring-legacy/src/test/java/io/micrometer/spring/web/servlet/MetricsFilterTest.java

```diff
@@ -103,33 +103,32 @@ public void subclassedTimedMethod() throws Exception {
 
         assertThat(this.registry.find("http.server.requests")
             .tags("status", "200", "uri", "/api/c1/metaTimed/{id}")
-            .value(Statistic.Count, 1.0).timer()).isPresent();
+            .timer().count()).isEqualTo(1L);
     }
 
     @Test
     public void untimedMethod() throws Exception {
         this.mvc.perform(get("/api/c1/untimed/10")).andExpect(status().isOk());
 
-        assertThat(this.registry.find("http.server.requests")
-            .tags("uri", "/api/c1/untimed/10").timer()).isEmpty();
+        assertThatThrownBy(() -> this.registry.find("http.server.requests")
+            .tags("uri", "/api/c1/untimed/10").timer())
+            .hasMessageContaining("Unable to locate a meter named 'http.server.requests'");
```

Comment:
Here is how we verify that metrics are missing. I could make the exception more strongly typed instead of just a long message if you like.